src/olympia/addons/tasks.py

```diff
@@ -243,7 +240,9 @@ def recreate_theme_previews(addon_ids, **kw):
             log.info('Recreating previews for theme: %s' % version.addon_id)
             xpi = get_filepath(version.all_files[0])
             theme_data = parse_addon(xpi, minimal=True).get('theme', {})
-            generate_static_theme_preview(theme_data, version.id)
+            generate_static_theme_preview.apply_async(
+                args=(theme_data, version.id), queue='adhoc'
```

Comment:
despite our warnings in settings_base, I guess this is okay @diox ?

Reply:
I guess so. It's worth trying out (though maybe confirm with ops if the queues are also separated on dev)

Reply:
ops confirmed the queues are separate on addons-dev too